code/game/objects/structures/watercloset.dm

```diff
@@ -15,7 +15,36 @@
 	. = ..()
 	open = round(rand(0, 1))
 	update_icon()
-
+	
+/obj/structure/toilet/contraband/Initialize(mapload)
+	..()
+	if(!open && mapload && prob(10))
+		var/ctype = pick( /obj/item/crowbar,
+				/obj/item/assembly/flash/handheld,
```

Comment:
might be OP to have a direct weapon

Reply:
Fair enough. Which one should I remove, exactly? Kitchen knife, carrot knife, shank or tail club?

Reply:
There should be a generic random loot spawner for maintenance that you could use for this.

I would say remove the flash because as far as I know that does not spawn in maintenance unless you get it from a mech fabricator.

Reply:
I left it in toilets because there's a very low chance (10%) for it to spawn and perhaps 5 toilets each game, so there's a small chance for it to spawn.

Reply:
>There should be a generic random loot spawner for maintenance that you could use for this.

I don't think this should use the generic maints spawner roll.